app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -477,6 +484,35 @@ def parse_stack_resource(resource, group)
         return uid, new_result
       end
 
+      def parse_managed_image(image)
+        uid = image.id.downcase
+
+        resource_group = ResourceGroup.where(:name => image.resource_group.downcase).find do |rg|
```

Comment:
Because I would like to set that relationship in the database for the sake of ActiveRecord, if possible. Then we aren't parsing resource group information out of strings, but instead are using Rails methods.